sdk/src/manifest_store.rs

```diff
@@ -335,7 +335,9 @@ mod tests {
         assert!(manifest.time().is_some());
     }
 
-    #[actix::test]
+    #[cfg_attr(not(target_arch = "wasm32"), actix::test)]
+    #[cfg_attr(target_arch = "wasm32", wasm_bindgen_test)]
+    #[allow(dead_code)]
```

Comment:
Do we need to allow dead code if we are supporting both alternatives?

Reply:
Likely not. I'll try removing